ocaml-lsp-server/test/e2e/__tests__/textDocument-completion.test.ts

```diff
@@ -285,17 +285,17 @@ describe("textDocument/completion", () => {
 
   it("completes labels", async () => {
     openDocument(outdent`
-      let f = ListLabels.map 
+      let f = ListLabels.map
```

Comment:
Tests now fail because of this

Reply:
Currently, the completion tests aren't in a good shape. Let's fix them first before wasting too many cycles on this.